proto/src/determined/api/v1/api.proto

```diff
@@ -446,6 +446,16 @@ service Determined {
       tags: [ "Trials", "Experiments" ]
     };
   }
+  // Get the list of workloads for a trial.
+  rpc GetTrialWorkloads(GetTrialWorkloadsRequest)
+      returns (GetTrialWorkloadsResponse) {
+    option (google.api.http) = {
+      get: "/api/v1/trial/{trial_id}/workloads"
```

Comment:
post discussion in backend eng, should we just make the the `/metrics` endpoint?